demo/src/app/components/typeahead/overview/sections/customize-options-list/template.html

```diff
@@ -0,0 +1,41 @@
+<p>
+  As said above, the options list can contain any kind of item, and this is handy since selecting an option will commit the actual item to the data model, allowing to store complex objects.
+</p>
+
+<p>
+  But the requirement is to be able to display those options, be it in the list of the popup or inside the input field.
+</p>
+
+
+
+<h5>Simple representation with a string</h5>
```

Comment:
Using result formatter ?

Reply:
Well, I say that very early in the first paragraph of the section. To me using `resultFormatter` in the title does not make it obvious they will use a string to represent the option.

With the two paragraphs names, I highlight the fact it's "string" vs "template" methods, no matter how the API is actually named ("formatter" is already a confusing name).